.chloggen/hostmetricsreceiver-mute-errors.yml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: hostmetricsreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add ability to mute errors (mainly due to access rights) coming from process scraper of the hostmetricsreceiver
```

```suggestion
note: Add ability to mute all errors (mainly due to access rights) coming from process scraper of the hostmetricsreceiver
```
Nit: Since there are already options to mute specific errors, we can clarify that this new option will mute _all_ errors.